apps/taiga-lumbermill/src/dashboards/register/register.component.html

```diff
@@ -0,0 +1,160 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+    class="card"
+>
+    <header tuiHeader>
+        <h2
+            tuiTitle
+            [style.text-align]="'center'"
+        >
+            Sign up
+        </h2>
+    </header>
+    <form
+        [formGroup]="form"
+        (submit)="onSubmit()"
+    >
+        <tui-input
+            formControlName="email"
+            type="email"
+            class="block-input"
+            [readOnly]="submitLoader()"
```

Comment:
form should not be readonly, it should be completely disabled. You can use `this.form.disable()' right in onSubmit handler.